cns/errorcodeframework/error.go

```diff
@@ -0,0 +1,44 @@
+package errorcodeframework
```

Comment:
I do like the suggestion provided by @ramiro-gamarra around the standard Error framework design across Azure Core. 

The structure: 
CNSResponse{
  "HTTPStatus": <Standard HTTP status code of success or failure>
"ResponseJson": 
  "error": {
    "code": "InvalidPasswordFormat",
    "message": "Human-readable description",
    "target": "target of error",
    "innererror": {
      "code": "PasswordTooShort",
      "minLength": 6,
    }
  }
}